source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -145,10 +156,64 @@ corresponding code.
          :end-before: end-change-stream-pipeline
          :language: csharp
 
-To learn more about modifying your change stream output, see the
-:manual:`Modify Change Stream Output
-</changeStreams/#modify-change-stream-output>` section in the {+mdb-server+}
-manual.
+Split Large Change Events Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your application generates change events that exceed 16 MB in size, the
+{+driver-short+} returns an error. To avoid this error, you can use
```

Comment:
Nit: Do we know the type of error it returns?

Reply:
The server error is BSONObjectTooLarge, I'll include that